packages/nx-web-ext/src/generators/application/generator.ts

```diff
@@ -66,10 +64,10 @@ export default async function (tree: Tree, options: NxWebExtGeneratorSchema) {
 
   switch (normalizedOptions.framework) {
     case 'angular':
-      await angularApp(tree, normalizedOptions);
+      await import('./sub-generators/angular').then(({ angularApp }) => angularApp(tree, normalizedOptions));
```

Comment:
slight mod: could we switch this entirely to async/await for both?

```ts
const { angularApp } = await import('./sub-generators/angular');
await angularApp(tree, normalizedOptions);
```

Reply:
yes sure